lexer/src/main/antlr4/gov/nasa/pds/api/registry/lexer/Search.g4

```diff
@@ -1,8 +1,9 @@
 grammar Search;
 
 query : queryTerm EOF ; 
-queryTerm : comparison | likeComparison | group ;
+queryTerm : comparison | likeComparison | existence | group ;
 group : NOT? LPAREN expression RPAREN ;
+existence : ( FIELD | STRINGVAL ) EXISTS ;
```

Comment:
I understand we want to be flexible, but I would only consider FIELD so to be more readable/simple on how the syntax is handled.
I mean it is always a field expected here, either it can have "" around of not is another topic that we don't need to spend time on now.

Reply:
We can ignore this comment, StringVal is used for REGEX. I did not get it first.